src/workerd/io/compatibility-date.capnp

```diff
@@ -1025,4 +1025,20 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # a temporary error on email delivery.
   # Note that by enabling this, user-provided Message-IDs are stripped and
   # Email Workers will generate and use its own.
+
+  removeNodejsCompatEOLv22 @117 :Bool
+      $compatEnableFlag("remove_nodejs_compat_eol_v22")
+      $compatDisableFlag("add_nodejs_compat_eol_v22")
+      $impliedByAfterDate(name = "removeNodejsCompatEOL", date = "2027-04-30");
```

Comment:
Do we want to include some padding to the EOL? 
We will keep those functions around in workerd anyway so adding i.e. 6 months of padding would not make a diff for workerd but might avoid our users running into troubles?